### ISSUE TITLE
USA spy drone reveals stealth units for all players 

### ISSUE THREAD

**Issue by leex279** [TheSuperHackers/GeneralsGamePatch#608]
The USA drone reveals all stealth units for all players instead of just for you/your team.

**Comment by leex279** [comment]
Lotus etc also do that --> maybe to be discussed?!

**Comment by commy2** [comment]
I think it is just a matter of being revealed means being revealed for everybody in FFA.

**Comment by leex279** [comment]
Yes, but why should other players see what I reveal? Think thats wrong. Does in real live the USA send their drone data to all other countries? :) Dont think so.

**Comment by commy2** [comment]
I don't think the game keeps a list of units each player has revealed. The unit becomes revealed and revealed units are shown. That's likely the extend of what the game does. Will have to change that in Thyme, but idk how feasible that is.
